domains/certificates/Query.API/API/GranularCertificateIssuer/WalletException.cs

```diff
@@ -9,8 +8,4 @@ public class WalletException : Exception
     public WalletException(string message) : base(message)
     {
     }
-
-    protected WalletException(SerializationInfo info, StreamingContext context) : base(info, context)
```

Comment:
Why has this been removed? It was not used or?

Reply:
It is a deprecated way, of serialization. as of dotnet 8, using that syntax, it is now obsolete. 
Adding it back produces a warning which, fails pipeline checks:

![image](https://github.com/Energinet-DataHub/energy-origin/assets/55732914/132f97d8-0cbb-46ac-9641-12b10799deb7)

Reply:
https://learn.microsoft.com/en-us/dotnet/fundamentals/syslib-diagnostics/syslib0051